webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala

```diff
@@ -176,14 +172,16 @@ trait ProjectsResponderADM {
   ): Task[ProjectOperationResponseADM]
 
   def projectDataGetRequestADM(id: ProjectIdentifierADM, user: UserADM): Task[ProjectDataGetResponseADM]
+
 }
 
 final case class ProjectsResponderADMLive(
```

Comment:
I'm wondering if the `ProjectsResponderADMLive` should go into its own file? I would vote for yes.